yarn.lock

```diff
@@ -6889,10 +6889,6 @@ grunt-sass@^3.0.2:
   resolved "https://registry.yarnpkg.com/grunt-sass/-/grunt-sass-3.1.0.tgz#a5936cc2a80ec08092d9f31c101dc307d1e4f71c"
   integrity sha512-90s27H7FoCDcA8C8+R0GwC+ntYD3lG6S/jqcavWm3bn9RiJTmSfOvfbFa1PXx4NbBWuiGQMLfQTj/JvvqT5w6A==
 
-grunt-search@^0.1.8:
```

Comment:
These changes [already removed](https://github.com/okta/okta-signin-widget/commit/a16fd481990a90e02245d0c726807d288fcdd535#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de) from `master` - which `5.5` hasn't backported just yet. Can you try rebasing?

Reply:
I know how this happened, I copied the yarn.lock from master (to revert some bad changes in the lock file). I'll copy the one from 5.5 instead and then there should be no change in this PR.